dev/openchpl_soft-delete.sql

```diff
@@ -13,7 +13,7 @@ CREATE TRIGGER developer_soft_delete AFTER UPDATE of deleted on openchpl.vendor
 
 CREATE OR REPLACE FUNCTION openchpl.certified_product_soft_delete()
 RETURNS TRIGGER AS $$
-BEGIN
+BEGINopenc
```

Comment:
Is this a typo?

Reply:
Sure looks that way - should be fixed now. Thanks!